src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appId__/config/com.adobe.cq.commerce.core.components.internal.services.UrlProviderImpl.config

```diff
@@ -1,6 +1,6 @@
-productUrlTemplate="${page}.${url_key}.html#${variant_sku}"
+productUrlTemplate="{{page}}.{{url_key}}.html#{{variant_sku}}"
 productIdentifierLocation="SELECTOR"
 productIdentifierType="URL_KEY"
-categoryUrlTemplate="${page}.${id}.html/${url_path}"
+categoryUrlTemplate="{{page}}.{{id}}.html/{{url_path}}"
```

Comment:
I'm not sure but I think @mhaack removed or disabled the Sling mappings, so we should maybe remove the suffix from the category URLs, or introduce a dedicated config for author and publish and only configure the susfix on publish.

Reply:
Correct the sample Sling Mappings are not part of the AEM Archetype, only of the https://github.com/adobe/aem-cif-guides-venia project.